src/locales/hu.json

```diff
@@ -1,13 +1,35 @@
 {
     "App": {
         "Notifications": {
+            "DependencyDescription": "Ez a verzió {name} nem támogat minden Mainsail funkciót. Frissítsd az újra {neededVersion} a jelenlegit {name}.",
+            "DependencyName": "Függőség: {name}",
+            "DismissAll": "Mind elvetése",
             "KlipperWarnings": {
                 "DeprecatedOption": "'{section}' / '{option}' opcióját leírtuk, és a következő verzióban már nem lesz benne.",
-                "DeprecatedValue": "'{section}' / '{option}' / Value '{value}' opcióját leírtuk, és a következő verzióban már nem lesz benne."
-            }
+                "DeprecatedOptionHeadline": "Elavult Klipper Opció",
+                "DeprecatedValue": "'{section}' / '{option}' / Value '{value}' opcióját leírtuk, és a következő verzióban már nem lesz benne.",
+                "DeprecatedValueHeadline": "Elavult Klipper Érték",
+                "KlipperWarning": "Klipper figyelmeztetés"
+            },
+            "MoonrakerWarnings": {
+                "MoonrakerComponent": "Moonraker: {component}",
+                "MoonrakerFailedComponentDescription": "Hiba a betöltés során, Moonraker '{component}'. Kérlek ellenőrizd a logokat, és javítsd a hibát.",
+                "MoonrakerWarning": "Moonraker figyelmeztetés",
+                "UnparsedConfigOption": "A [{section}] szekcióban helytelen konfig opciót '{option}: {value}' detektáltam. Ez lehet egy már nem elérhető opció, vagy egy hibásan betöltődő modul is okozhatja. A jövőben ez egy indítási hibaként fog jelentkezni.",
+                "UnparsedConfigSection": "Helytelen konfig szekciót [{section}] detektáltam. Ezt egy egy hibásan betöltődő komponens okozhatja. A jövőben ez egy indítási hibaként fog jelentkezni."
+            },
+            "Never": "Soha",
+            "NextReboot": "Következő reboot",
+            "NoNotification": "Értesítés nem elérhető",
+            "Remind": "Emlékeztető:"
+        },
+        "NumberInput": {
+            "GreaterOrEqualError": "Nagyobbnak vagy egyenlőnek kell lennie mint {min}!",
+            "MustBeBetweenError": "A {min} és {max} között kell lennie!",
+            "NoEmptyAllowedError": "Nem lehet üres a bemenet!"
         },
         "Printers": "Nyomtatók",
-        "ThrottledStates": {
+        "ThrottledStates": {ű
```

Comment:
The `ű` character does not belong to here

Reply:
Agreed